src/Ng/CommandHelpers.cs

```diff
@@ -140,11 +236,7 @@ public static bool GetVerbose(IDictionary<string, string> arguments)
         public static bool GetBool(IDictionary<string, string> arguments, string argumentName, bool defaultValue)
         {
             string argumentValue;
-            if (!arguments.TryGetValue("-" + argumentName, out argumentValue))
-            {
-                return defaultValue;
-            }
-
+            TryGetArgument(arguments, argumentName, out argumentValue);
             return string.IsNullOrEmpty(argumentValue) ? defaultValue : argumentValue.Equals("true", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
>argumentValue.Equals("true", StringComparison.InvariantCultureIgnoreCase); [](start = 72, length = 74)

bool.Parse. In case the user misspelled falce, we want to fail